docs/4.2/features/ssh_pam.md

```diff
@@ -229,11 +229,9 @@ a shell for said user.
 ### Examples
 **Using pam_exec.so**
 
-Using `pam_exec.so` is the easiest way to use the PAM stack to create a user if the
-user does not already exist. The advantage of using `pam_exec.so` is that it usually
-ships with the operating system. The downside is that it doesn't provide access to
-some additional environment variables that Teleport sets (see the `pam_script.so`
-example for those) to use additional identity metadata in the user creation process.
+Using `pam_exec.so` is the easiest way to use the PAM stack to create a user if
+the user does not already exist. `pam_exec.so` usually ships with the operating
+system.
```

Comment:
By usually, does this mean all modern version Ubunutu, Debian, CentOS and RHEL support this?

Reply:
It existed in linux-pam since 2006: https://github.com/linux-pam/linux-pam/blob/master/ChangeLog-CVS#L4443-L4457
So yes, it should come bundled with any default PAM setup.